CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [5.0.1] - 2017-05-09
```

Comment:
I'm not sure I understand what you want to have. Should I remove the entire added content of the changelog, just the heading, or is there actually something missing?
It doesn't matter to me if it takes some time to be integrated into the release version, since this should be a rare occurence.